src/Simplex/Chat.hs

```diff
@@ -6014,13 +6118,29 @@ sameGroupProfileInfo p p' = p {groupPreferences = Nothing} == p' {groupPreferenc
 
 createInternalChatItem :: forall c d m. (ChatTypeI c, MsgDirectionI d, ChatMonad m) => User -> ChatDirection c d -> CIContent d -> Maybe UTCTime -> m ()
 createInternalChatItem user cd content itemTs_ = do
+  ci <- createInternalChatItem_ user cd content itemTs_
+  toView $ CRNewChatItem user (AChatItem (chatTypeI @c) (msgDirection @d) (toChatInfo cd) ci)
+
+createInternalChatItem_ :: (MsgDirectionI d, ChatMonad m) => User -> ChatDirection c d -> CIContent d -> Maybe UTCTime -> m (ChatItem c d)
```

Comment:
why both createInternalChatItem_ and createLocalChatItem are required? probably refactoring of the former can be reverted. so there's only createInternalChatItem without underscore and createLocalChatItem